flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/config/Environment.java

```diff
@@ -50,6 +51,9 @@
 
 	public static final String DEPLOYMENT_ENTRY = "deployment";
 
+	public static final String[] CONFIGURATION_ENTRIES = new String[]{
```

Comment:
I think a prefix of `table` is enough. If we add more options we don't need to change this location again.

Reply:
![got-it](https://github.com/TsReaper/stickers/blob/master/stickers/eastern-timber-wolf/got-it.jpg?raw=true)